### PR TITLE
Updated serializer version constraint

### PR DESCRIPTION
**Pull request by fieg**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "require": {
     "php": ">=5.5",
     "symfony/symfony": "^2.6",
-    "jms/serializer": "^1.0"
+    "jms/serializer-bundle": "^0.13|^1.0"
```
Comment (pkruithof) [inline composer.json]: Why change this to the bundle? The dependency is only on the library.

Comment (fieg) [inline composer.json]: It's using a service definition provided by the bundle:
https://github.com/treehouselabs/base-api-bundle/blob/master/src/TreeHouse/BaseApiBundle/Controller/BaseApiController.php#L237

Comment (pkruithof) [inline composer.json]: Good catch!

**Comment by pkruithof** [comment]
Can you make sure tests are being done with both the lowest and highest dependencies?

**Pull request merged**